common/utils/analytics.js

```diff
@@ -93,6 +93,14 @@ export default ({ category, contentType, pageState, featuresCohort }: Props) =>
     window.performance.clearMeasures();
   };
 
+  setTimeout(() => {
+    ReactGA.event({
+      category: 'Engagement',
+      action: 'Time on page >=',
+      label: '10 seconds'
+    });
+  }, 10000);
```

Comment:
Don't mind, but wondered what the logic of separating the label out was?
Why not have it all in the `action`? In terms of analysing, that might be easier?

Reply:
I'm easy, this is what @hthair suggested when I asked her (I think)